src/test/java/org/jabref/logic/importer/fetcher/transformators/JstorQueryTransformerTest.java

```diff
@@ -29,11 +36,25 @@ public String getTitlePrefix() {
 
     @Override
     public void convertYearField() throws Exception {
-
+        ArXivQueryTransformer transformer = ((ArXivQueryTransformer) getTransformer());
```

Comment:
Why is the ArXiv transformer here? Also the test seems to be duplicates of other tests...

Also it might be a good idea to make InfixTransformerTest generic so that getTransformer returns already the correct type of the transformer without explicit casting.